cla-backend-go/events/repository.go

```diff
@@ -848,7 +848,7 @@ func (repo *repository) AddDataToEvent(eventID, parentProjectSFID, projectSFID,
 	ue.AddAttributeValue(":company_sfid_project_id", &dynamodb.AttributeValue{S: aws.String(companySFIDProjectID)}, companySFID != "" && projectID != "")
 	ue.AddAttributeValue(":company_sfid_cla_group_id", &dynamodb.AttributeValue{S: aws.String(companySFIDClaGroupID)}, companySFID != "" && claGroupID != "")
 
-	ue.AddUpdateExpression("#parent_project_sfid = :parent_project_sfid", parentProjectSFID != "")
+	ue.AddUpdateExpression("#foundation_sfid = :foundation_sfid", parentProjectSFID != "")
```

Comment:
@umeshlumbhani247 how does this update resolve the update crash issue ?

Reply:
It was getting crash error like:
unable to add extra details to event : {event_id} . error = ValidationException: Invalid UpdateExpression: An expression attribute value used in expression is not defined; attribute value: :parent_project_sfid
	status code: 400, request id: CN06B95SAUMMA4EI9OE9FONEABVV4KQNSO5AEMVJF66Q9ASUAAJG

we have to add foundation_sfid not parent_project_sfid.